src/_locales/de-DE.json

```diff
@@ -0,0 +1,94 @@
+{
+  "audio-device-communications": "System Default Communications",
+  "audio-device-default": "Standardeinstellung",
+  "audio-device-unknown": "Unbekanntes Gerät",
```

Comment:
@MCManuelLP rather "Standardgerät" and "Standardkommunikationsgerät" . Would line up perfectly with MS's naming